.github/workflows/analyze.yml

```diff
@@ -61,11 +62,11 @@ jobs:
 
   analyze-python-scripts:
     name: Python Scripts
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Please leave 22.04 for stability reasons, we should better upgrade it manually once 24.04 is released and tested.

Reply:
655d58a24595e868112db1cf603572d10e633b80